src/commands/core/follow.ts

```diff
@@ -0,0 +1,19 @@
+import Command from '../../lib/structures/Command';
+import { TypicalGuildMessage } from '../../lib/types/typicalbot';
+import { Modes, PermissionsLevels } from '../../lib/utils/constants';
+
+export default class extends Command {
+    mode = Modes.STRICT;
+    permission = PermissionsLevels.SERVER_ADMINISTRATOR;
+
+    async execute(message: TypicalGuildMessage, parameters: string) {
+        const [type, id] = parameters.split(' ');
+        const channel = message.mentions.channels.first() ?? message.guild.channels.cache.get(id) ?? message.channel;
+        const isStatus = type?.toLowerCase() === 'status';
+
+        // @ts-ignore
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        this.client.api.channels(isStatus ? '621817852726607882' : '268559149175013376').followers.post({ data: { webhook_channel_id: channel.id } });
```

Comment:
```suggestion
        await this.client.api.channels(isStatus ? '621817852726607882' : '268559149175013376').followers.post({ data: { webhook_channel_id: channel.id } });
```

Reply:
This should also check to ensure that the bot has Manage Webhooks permission and send an error to the user if not.